frontend/src/pages/ui/TopicInfoModalButton.tsx

```diff
@@ -33,11 +32,12 @@ export default function TopicInfoModalButton() {
   if (!configArray) return <></>
 
   return (
+    // @BEN: keep this as fully MUI component?
```

Comment:
fine for this PR, yep. just remove this comment and merge as is. If we find this exact type of component elsewhere and it makes sense we can abstract into a `<HetMoreInfoButton>` or something but i dont want to prematurely optimize

Reply:
ok, understood